### PR TITLE
updated the nav bar to be properly responsive, making text size persistent too

### PR DESCRIPTION
**Pull request by Pomax**
The navigation bar should now be properly responsive, for resolutions as low as 400px, and text size settings form the nav bar should be persistent

### CODE REVIEW

File: js/fc/ui/text.js [hunk 1]
```diff
@@ -4,9 +4,20 @@ define(function() {
...
     var codeMirror = options.codeMirror;
-    
     var tzo = $("#text-size-options");
-    
+
+    /**
+     * Check is local storage is supported, and if so, whether
+     * the font size has already been stored previously.
+     */
+    var supports_local_storage = function() {
```
Comment (brianloveswords) [inline js/fc/ui/text.js]: Nit: Stylewise, this should be probably be `supportsLocalStorage`.

File: css/editor.css [hunk 2]
```diff
@@ -105,6 +104,61 @@ img {
...
+    max-width: 999px !important;
+    width: auto !important;
+    padding-right: 1em;
+    color: black;
+  }
+  #nav-options .nav-item {
+    border: none;
+  }
+  #editor-nav-item {
+    border-right: 1px solid #D6D6D6 !important;
```
Comment (brianloveswords) [inline css/editor.css:125]: Better to use a higher specificity than to use `!important`.

Comment (Pomax) [inline css/editor.css:125]: As a media size overwrite, there are no "more specific" selectors - it's the same selectors, but they have to get redefined the smaller we get. In this case if there is a more specific selector, it also needs to be used in the normal CSS.

File: js/fc/ui/text.js (hunk 1 above)
Comment (Pomax) [inline js/fc/ui/text.js]: agreed. updated in https://github.com/Pomax/friendlycode/commit/7d41a7a9754019e3cf7558add37f86c9c3a59df8

File: css/editor.css (hunk 2 above)
Comment (brianloveswords) [inline css/editor.css:125]: Ah okay. It's fine then since it's only used in the media queries.

**Comment by brianloveswords** [comment]
Code looks good. Checked out the branch and it works great on Firefox 14, Safari 5.1.7 and Chrome 19.0.

**Pull request merged**